templates/careers/travel.html

```diff
@@ -1,66 +1,112 @@
+      <div class="col-6">
+        <p>Canonical is unlike any other company in the world. It is amazing for people who are organized, passionate about technology, and like to travel. People at Canonical say the people at Canonical are amazing. That's special.</p>
+        <p class="p-heading--5">We're a technology company</p>
+        <p>Every role at Canonical is a tech role. We have a high expectation of competence and passion for technology, regardless of your title. The future is already here, as open source, and Canonical's mission is to deliver it to the world. We are part of the biggest change in technology history &ndash; to open source &ndash; and we play a critical role in broadening the benefits of open source to ever more people. Every role at the company involves good judgment against the backdrop of the fast-moving technology landscape. We look for people who find that interesting.</p>
+        <p class="p-heading--5">Second, we're a fully distributed organisation.</p>
+        <p>Most people work from home, using the latest communications technology to be productive and maintain a high level of coordination with colleagues. Digital collaboration is how we enable people to enjoy their home environment AND work with the best in the world in their domain.</p> 
+        <p>Zero commute means more time for the things you enjoy, it also means you need to be organized and effective in a distributed workplace. You get tremendous, perhaps unprecedented, flexibility in how you organise your day. At the same time, you have to be productive and a good counterpart to colleagues around the world. You need to be a team player, and you need to deliver what you promise, because teams depend on each other and don’t have daily eyes on one another. We have very little office politics but we do have very high expectations of commitment and independent execution.</p>   
+        <p>We also travel more than most companies, because our customers and our teams are global, and we need regular face-time with both.</p>  
+        <p class="p-heading--5">Third, we aspire to lead on the global stage.</p>   
+        <p>There's no other way to describe it than hard &ndash; hard work, hard challenges, hard competition. Most of us are avid students of greatness &ndash; we are interested in how the latest things work, how the best organizations work, how the smartest companies work, and we aim to hold our own in that crowd.</p>
+        <p>You will need to show that you can compete in deeply challenging intellectual fields. Your colleagues are aiming for the top, and they depend on you to set the same standard in your area of responsibility. That's tough. It's also incredibly satisfying, with continuous learning and challenge. You will have exceptional opportunities and exceptional colleagues. They will expect you to be exceptional too.</p>
+      </div>
+    </div>
+  </div>  
+</section>
+<script defer src="{{ versioned_static('js/sprint-map.js') }}"></script>
```

Comment:
Do you think it is worth adding this to the build?

Reply:
I'm actually not sure. My first instinct was no because this isn't building something based on a dependency (like fuse js or latest news for example), but there's a very high likelihood that I'm totally wrong in that assessment 😅 @anthonydillon what would be best practice here?